_data/committers.yml

```diff
@@ -287,11 +303,15 @@
 - name: Deepak Majeti
   role: Committer
   alias: mdeepak
-  affiliation: Ahana
+  affiliation: IBM
 - name: Dominik Moritz
   role: Committer
   alias: domoritz
   affiliation: Carnegie Mellon University
+- name: Ed Seidl
+  role: Committer
+  alias: etseidl
+  affiliation: "LLNL"
```

Comment:
I wonder if there is a reason there are quotes on this one. Like why not  the following 🤔 
```suggestion
  affiliation: LLNL
```

Reply:
I think that was just due to it being "-" (which is necessary to avoid yaml 'fun') before and Ed just added it in the quotes.

Reply:
And it doesn't change the rendered result :)
![image](https://github.com/user-attachments/assets/46c12d25-2677-4289-8947-e24d61d276e9)